src/option.h

```diff
@@ -37,7 +37,7 @@ namespace librealsense
         std::vector<std::function<void(float)>> _callbacks;
     };
 
-    class readonly_option : public option
+    class readonly_option : virtual public option
```

Comment:
Diamond inheritance?

Reply:
It is because I inherit both from readonly and from uvc_xu_option, and they both inherit from option.
So, in order to avoid ambiguity, had to add virtual to both (multiple inheritance, diamond problem - see https://www.geeksforgeeks.org/multiple-inheritance-in-c/ for example)